src/sugar3/graphics/radiopalette.py

```diff
@@ -36,24 +123,49 @@ def __init__(self, **kwargs):
         self.connect('notify::palette', self.__palette_cb)
 
     def __palette_cb(self, widget, pspec):
+        '''
+        A callback function
+        it returns if the "self.props.palette" is an instance of "RadioPalette",
+        if it is not, then it updates the button with the function "update_button()"
+        '''
         if not isinstance(self.props.palette, RadioPalette):
             return
         self.props.palette.update_button()
 
 
 class RadioToolsButton(RadioMenuButton):
+    '''
+    This is a button used to open a group of widgets in a RadioPalette(menu)
+    It is essentially just a RadioMenuButton but with the added function
+    do_clicked(self), which performs the given lines of code.    
+    Example Usage: button = RadioToolButton(icon_name='document-save',
+                                            group=group)
```

Comment:
Can you please document the icon_name and group as "Kwargs:" parameters?  Saying what type they must be will help readers.